test/fakeintake/client/client_test.go

```diff
@@ -217,4 +220,19 @@ func TestClient(t *testing.T) {
 		err := client.FlushServerAndResetAggregators()
 		assert.NoError(t, err)
 	})
+
+	t.Run("getFlare", func(t *testing.T) {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
Nice !